runtime/onert/backend/cpu/ops/AddLayer.cc

```diff
@@ -32,22 +32,21 @@ void AddLayer::addFloat32()
   float output_activation_min = 0, output_activation_max = 0;
   CalculateActivationRange(_activation, &output_activation_min, &output_activation_max);
   nnfw::cker::BinaryArithmeticOpParam op_params;
-  op_params.type = nnfw::cker::BinaryArithmeticOpType::ADD;
   op_params.float_activation_max = output_activation_max;
   op_params.float_activation_min = output_activation_min;
 
   const bool need_broadcast =
       nnfw::cker::ProcessBroadcastShapes(getTensorShape(_lhs), getTensorShape(_rhs), &op_params);
   if (need_broadcast)
   {
-    nnfw::cker::BroadcastBinaryArithmeticOp<float>(
+    nnfw::cker::BroadcastBinaryArithmeticOp<nnfw::cker::BinaryArithmeticOpType::ADD>(
```

Comment:
I think it is better to specifiy variable type in here. 
```suggestion
    nnfw::cker::BroadcastBinaryArithmeticOp<nnfw::cker::BinaryArithmeticOpType::ADD, float>(
```